base/essentials.jl

```diff
@@ -417,9 +430,11 @@ esc(@nospecialize(e)) = Expr(:escape, e)
 
 Annotates the expression `blk` as a bounds checking block, allowing it to be elided by [`@inbounds`](@ref).
 
-Note that the function in which `@boundscheck` is written must be inlined into
-its caller with [`@inline`](@ref) in order for `@inbounds` to have effect.
+!!! note
+    The function in which `@boundscheck` is written must be inlined into
+    its caller with [`@inline`](@ref) in order for `@inbounds` to have effect.
```

Comment:
Is this really true? If the compiler inlines the function without `@inline`, won't the `@boundscheck` block be elided?

Reply:
Correct, this claim is wrong: inlining for any reason is sufficient.

Reply:
Should I just delete this `!!! note` and everything indented then?

Reply:
Just delete the `with [@inline](@ref)` part.